src/LogViewer/chunk-worker.js

```diff
@@ -128,14 +128,21 @@ const update = (response) => {
 const error = () => self.postMessage(JSON.stringify({ type: 'error' }));
 const loadEnd = () => self.postMessage(JSON.stringify({ type: 'loadend' }));
 
-const toHtml = (index) => {
+const getParagraphClass = (lineNumber, metadata) => {
+  const { highlightStart, highlightEnd } = metadata;
+
+  return lineNumber >= highlightStart && lineNumber <= highlightEnd ? 'highlight' : '';
```

Comment:
As mentioned, change this function to return the whole paragraph attribute so we can reduce unnecessary string creation:

```js
return lineNumber >= highlightStart && lineNumber <= highlightEnd ?
  ' class="highlight"' :
  '';
```